src/internal/tester/cli.go

```diff
@@ -32,34 +29,3 @@ func StubRootCmd(args ...string) *cobra.Command {
 
 	return c
 }
-
-func NewContext() (context.Context, func()) {
```

Comment:
moved because I keep forgetting what file these are in and always try to look for `/tester/context.go`